src/skeptic/lib.rs

```diff
@@ -777,10 +785,12 @@ pub mod rt {
         let mut deps_dir = target_dir.clone();
         deps_dir.push("deps");
 
+        let edition = get_edition(&root_dir).expect("failed to read Cargo.toml");
         let mut cmd = Command::new(rustc);
         cmd.arg(in_path)
             .arg("--verbose")
             .arg("--crate-type=bin")
+            .arg(format!("--edition={}", edition))
```

Comment:
the `--edition` is added even if there is no edition found. I'd make this conditional because this makes older rustc versions unhappy https://travis-ci.org/budziq/rust-skeptic/jobs/462642234 :(

we can easily fix this backwards compatibly

Reply:
Ah, sure. I had originally set it up so that if the edition is 2015 (the default value) it wouldn't add anything, but then I was having trouble testing it (the skeptic test suite doesn't use 2018 at all...).